sceptre/helpers.py

```diff
@@ -117,3 +120,10 @@ def null_context():
     available in py3.6, so providing it here instead.
     """
     yield
+
+
+def get_response_datetime(resp):
+    try:
+        return datetime(*eut.parsedate(resp["ResponseMetadata"]["HTTPHeaders"]["date"])[:6])
```

Comment:
I think a comment here about why you're only selecting the first 6 characters would be appropriate. What's an example of the format this header is in?

Reply:
Fair enough. Can do.